src/views/Wiki.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <main class="wiki container">
+    <article id="article">
+      <router-view />
+    </article>
+    <aside>
+      <h1>Wiki</h1>
+      <sidebar />
+    </aside>
+  </main>
+</template>
+
+<script>
+  import Sidebar from '../wiki/_Sidebar.md';
+
+  export default {
+    components: {
+      Sidebar
+    },
+  }
+</script>
+
+<style lang="scss">
+  .wiki {
+    display: flex;
+    align-items: flex-start;
+	  background-color: $grey;
+    height: 100%;
+    flex-wrap: wrap;
+    
+    .header-anchor {
+      text-decoration: none;
+      display: none;
+    }
+  }
+
+  aside {
+    width: 100%;
+    padding: 2em 2em 0 2em;
+	  background-color: rgba(0,0,0,.25);
+    order: 2;
+
+    @include breakpoint($md) {
+      flex: 2;
+      order: 1;
+      overflow: auto;
+      height: 100%;
+    }
+
+    @include breakpoint($lg) {
+      flex: 0 0 24em;
+    }
+
+    section {
+      margin-bottom: 2rem;
+    }
+
+    h1 {
+      font-size: 2rem;
+      margin-top: 0;
+    }
+
+    a {
+      text-decoration: none;
```

Comment:
```suggestion
      text-decoration: none;
      margin: .2rem;
```

To have the outlines show better :)